src/platform_impl/windows/dark_mode.rs

```diff
@@ -141,6 +150,7 @@ fn set_dark_mode_for_window(hwnd: HWND, is_dark_mode: bool) -> bool {
       };
 
       let status = set_window_composition_attribute(hwnd, &mut data as *mut _);
+      set_preferred_app_mode(is_dark_mode);
```

Comment:
It should handle the return value as well, right?

Reply:
I found that the `SetPreferredAppMode` function returns the previous value that was set which will always start at 0 (`PreferredAppMode::Default`) when the application first runs, I don't really know if checking this value is relevant.

I do not have a version of Windows 10 to test the older `AllowDarkModeForApp`.

Removing the return value from `set_preferred_app_mode` might make more sense.